pkgs/by-name/sw/switch-to-configuration-ng/src/src/main.rs

```diff
@@ -1777,16 +1745,13 @@ won't take effect until you reboot the system.
     // time because we'll get a "Failed to add path to set" error from systemd.
     let units_to_start_filtered = filter_units(&units_to_filter, &units_to_start);
     if !units_to_start_filtered.is_empty() {
-        let mut units = units_to_start_filtered
-            .keys()
-            .into_iter()
-            .map(String::as_str)
-            .collect::<Vec<&str>>();
-        units.sort_by_key(|name| name.to_lowercase());
-        eprintln!("starting the following units: {}", units.join(", "));
+        eprintln!(
+            "starting the following units: {}",
+            format_unit_set(&units_to_start_filtered)
+        );
     }
 
-    for unit in units_to_start.keys() {
+    for unit in &units_to_start {
```

Comment:
Ah, but oddly enough, this one discarded the sorting and `to_lowercase`'ing before this PR.... Huh.